scripts/js/lib/api/processHtml.ts

```diff
@@ -206,6 +208,23 @@ export function addLanguageClassToCodeBlocks(
   });
 }
 
+export function updateRedirectedExternalLinks(
+  $: CheerioAPI,
+  $main: Cheerio<any>,
+  redirects: { [old: string]: string },
+): void {
+  $main.find("a").each((_, a) => {
+    const $a = $(a);
+    const href = $a.attr("href");
+    const replacement = href && redirects[href];
+    if (replacement) {
+      $a.attr("href", replacement);
+      const newText = $a.text().replaceAll(href, replacement);
+      $a.text(newText)
```

Comment:
This should require an exact match, not partial

Reply:
Really? What if the link text was `https://google.com (opens in new tab)` or similar?